src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($squareSize + 1) * 7 + 20px;
```

Comment:
what is 20px? maybe we can make it variable?

Reply:
agree, add `20px` to a variable